### PR TITLE
Documentation Updates

### PR DESCRIPTION
**Pull request by hu3man**
This PR:
- Adds a scaffold-basics page
- Adds docs for taco-shop scaffold
- Adds docs for hello-tacos-tutorial
- Adds docs for quickstart scaffold
- Removes references to Taqueria state and the @taqueria/state page
- Adds environment docs
- Adds docs for the github action
- Adds release notes
- Refactors docs for current schema and ghostnet
- Updates the installation

### CODE REVIEW

File: website/docs/config/environments.mdx [hunk 1]
```diff
@@ -1,3 +1,83 @@
...
+
+The sandboxes property holds a comma separated list of named sandboxes. The sandboxes referenced must be configured in the `sandbox` property of `config.json`. You can add zero or more sandboxes to an environment
+
+### The `storage` Property
+
+:::caution
+You must declare initial storage values for each contract you are originating. Trying to originate a contract to an environment that does not have a storage value for that contract will result in an error
+:::
+
+The storage property holds an array of initial storage values that will be used to originate each smart contract. You must define an initial storage for each `.tz` file that you will be originating
```
Comment (mweichert) [inline website/docs/config/environments.mdx:64]: The storage property is a key/value mapping.

As such, I'd suggest:
```
The storage property is a key/value mapping, with the key being the name of the contract, and the value being the initial storage value. 

The initial storage value may be specified as any JSON literal value, a string containing a Michelson expression, or a JSON-encoded Michelson expression.
```

File: website/docs/release-notes.mdx [hunk 2]
```diff
@@ -2,6 +2,33 @@
 title: Taqueria Releases
```
Comment (mweichert) [inline website/docs/release-notes.mdx:2]: Is this redundant with the changelog that Github provides?
https://github.com/ecadlabs/taqueria/releases

My only concern is that it adds a step to the deployment process, and is another thing to maintain.

**Review by mweichert — COMMENTED** [review]
LGTM @hu3man -I left a few comments for your review.

**Review by russellmorton — APPROVED** [review]

File: website/docs/config/networks.mdx [hunk 3]
```diff
@@ -32,14 +32,13 @@ To configure and use a Taqueria network, you will require the following:
...
 
-
 ## Configuration
 
 :::note
 The network configuration for various protocols, as well as the faucet for each can be found at [teztnets.xyz](https://teztnets.xyz/).
 :::
 
-To configure a network for use, you will need to configure the network, and then add that network to an environment. To illustrate the configuration of a network, we will use the [ithaca testnet](https://teztnets.xyz/ithaca-testnet/) as an example in this document and will configure a network named `ithaca`, and an environment named `ithacanet`
+To configure a network for use, you will need to configure the network, and then add that network to an environment. To illustrate the configuration of a network, we will use [ghostnet](https://teztnets.xyz/ithaca-testnet/), a long running Tezos test network as an example in this document and will configure a network named `ithaca`, and an environment named `testing`
```
Comment (GImbrailo) [inline website/docs/config/networks.mdx:41]: > a long running Tezos test network as an example in this document and will configure a network named `ithaca`

Should this not also be `ghostnet`?

**Review by GImbrailo — APPROVED** [review]

**Pull request merged**